ontology_processing/graph_creation/make_graph.py

```diff
@@ -934,41 +81,4 @@ def makeGraph(onto_path, edge_path, output_folder_path="."):
     not_test_ont = get_non_test_ont()
     get_test_ontology(T, valid_test_ont, not_test_ont)
 
-    save_test_ontology_to_json(T, output_folder_path)
-
-
-def main(args):
-    """
-    Main function to make networkx graph object from reference ontology and edge list.
-
-    input: args = args from the argument parser for the function
-    (refOntologyPath, refEdgeListPath)
-    output: saves a python pickle file of the networkx object, and yaml and json of the networkx object
-
-    example: python3 make_graph.py "./climate_mind_ontology20200721.owl" "output.csv"
-    """
-
-    # set arguments
-    onto_path = args.refOntologyPath
-    edge_path = args.refEdgeListPath
-
-    # run makeGraph function
-    # Doesn't set path? I have to add default argument for path for this to work.
-    makeGraph(onto_path, edge_path)
-
-
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser(
```

Comment:
Would be nice if we could keep this command line option to be able to run this script from the command line with arguments accepted. What are your thoughts?